pkg/controller/windowsmachineconfig/nodeconfig/nodeconfig.go

```diff
@@ -209,39 +232,30 @@ func (nc *nodeConfig) handleCSR(requestorFilter string) error {
 	return nil
 }
 
-// GetNode returns a pointer to the node object associated with the external IP provided
-func (nc *nodeConfig) getNode(externalIP string) (*v1.Node, error) {
-	var matchedNode *v1.Node
-
+// getNode returns a pointer to the node object associated with the instance id provided
+func (nc *nodeConfig) getNode() error {
 	nodes, err := nc.k8sclientset.CoreV1().Nodes().List(metav1.ListOptions{})
```

Comment:
nit: Sorry, it's a miss on my side in earlier reviews but shouldn't you `List` only the Windows nodes? This way will limit the items in our `nodes` object.

Reply:
This is not a nit. Please fix.